ctera/src/main/java/ch/cyberduck/core/ctera/CteraSession.java

```diff
@@ -154,14 +220,42 @@ protected void logout() throws BackgroundException {
     @Override
     @SuppressWarnings("unchecked")
     public <T> T _getFeature(final Class<T> type) {
+        final CteraSession session = this;
         if(type == Touch.class) {
-            return (T) new CteraTouchFeature(this);
+            return (T) new CteraTouchFeature(this) {
```

Comment:
Extract inner class to upper.

Reply:
Done e57ba170367804bdddabca3d79878d9cc6a410ec